pkg/resource/aws/aws_ecr_repository_ext.go

```diff
@@ -0,0 +1,13 @@
+package aws
```

Comment:
Since there are no normalizations to do, you can remove this file, it is useless.

Reply:
agreed